package.json

```diff
@@ -89,6 +89,8 @@
     "publish:all": "npm publish dist/npm/clarity-ui; npm publish dist/npm/clarity-angular; npm publish dist/npm/clarity-icons",
     "presample-app": "npm i dist/npm/clarity-ui; npm i dist/npm/clarity-icons; npm i dist/npm/clarity-angular;",
     "sample-app": "gulp sample",
+    "preks:publish": "gulp npm:prepare --version $npm_package_version",
+    "ks:publish": "cd src/ks-app; ng build --preserve-symlinks --prod; cd ../../dist/ks-app; cat index.html > 404.html; surge; cd ../../",
     "cssTest": "./docker-ctrl/cdc.js -t",
```

Comment:
Just a note here, we'll need to update the `preks:publish` script here later, as we move off of gulp and onto webpack.